### ISSUE TITLE
Attribute classifiers

### ISSUE THREAD

**Issue by WangJYao** [wonjongg/StyleCariGAN#17]
How to train the attribute classifiers?
Where is the training code？
Thank you.

**Comment by wonjongg** [comment]
We trained the attribute classifier with WebCariA dataset by following their protocols.
The network architecture & loss function for training are ResNet-18 & binary cross-entropy loss in PyTorch (please see page 9 in our paper)
The training code for the attribute classifier is not included in this project. 

Thank you!
